native-activity/app/build.gradle

```diff
@@ -5,38 +5,39 @@ model {
         compileSdkVersion = 23
         buildToolsVersion = "23.0.2"
 
-        defaultConfig.with {
+        defaultConfig {
             applicationId = "com.example.native_activity"
             minSdkVersion.apiLevel    = 9
             targetSdkVersion.apiLevel = 23
         }
-    }
-
-    android.ndk {
-        moduleName ="native-activity"
-        CFlags.add("-I${file("src/main/jni/native_app_glue")}".toString())
-        ldLibs.addAll(["log", "android", "EGL", "GLESv1_CM"])
-    }
-    android.buildTypes {
-        release {
-            minifyEnabled = false
-            proguardFiles.add(file('proguard-rules.txt'))
-        }
-    }
-    android.productFlavors {
-        create ("arm7") {
-            ndk.abiFilters.add("armeabi-v7a")
+        ndk {
+            stl = "gnustl_static"
```

Comment:
maybe more explanation about parent project?  the build for each module inside the project is done separately ( right now ), so the parameters settings are not shareable.
for one module, we just have one stl_"gnustl_static" type.  it is not possible to share stl type between main project module and native-activity project (we put this one to a project later )